### PR TITLE
Resolved bug #119

### PR DESCRIPTION
**Pull request by ilmcconnell**

### CODE REVIEW

File: cosmos/ingestion/ingest/ingest.py [hunk 1]
```diff
@@ -35,7 +35,7 @@
 logging.getLogger("ingest.process.detection.src.torch_model.train.data_layer.xml_loader").setLevel(logging.ERROR)
 
 logger = logging.getLogger(__name__)
-logger.setLevel(logging.ERROR)
+logger.setLevel(logging.DEBUG)
```
Comment (iross) [inline cosmos/ingestion/ingest/ingest.py:38]: Does this make the logging overly verbose?

**Review by iross — CHANGES_REQUESTED** [review]
Looks good, other than the question re: logging level.

File: cosmos/ingestion/ingest/ingest.py (hunk 1 above)
Comment (ilmcconnell) [reply cosmos/ingestion/ingest/ingest.py]: Thanks @iross - a sloppy mistake on my part. I've changed it back to DEBUG.

**Review by iross — APPROVED** [review]
Thanks!

**Pull request merged**